pkg/util/client.go

```diff
@@ -27,3 +30,18 @@ func GetClientSet(file string) (clientset.Interface, error) {
 
 	return clientset.NewForConfig(restConfig)
 }
+
+// SplitSliceString is a helper function to handle StringSliceFlag containing mutliple values
+// By default, StringSliceFlag only supports repeated values, not multiple values
+// e.g. --foo="bar,car" --foo=baz will result in []string{"bar", "car". "baz"}
+func SplitSliceString(flag cli.StringSlice) []string {
+	result := []string{}
+	for _, s := range flag.Value() {
+		if strings.Contains(s, ",") {
+			result = append(result, strings.Split(s, ",")...)
+		} else {
+			result = append(result, s)
+		}
```

Comment:
`cli.StringSlice` is just a type alias for `[]string` so I think you can actually make this a lot simpler.
https://github.com/urfave/cli/blob/v1.22.12/flag_string_slice.go#L10

```suggestion
func SplitSliceString(ss []string) []string {
	result := []string{}
	for _, s := range ss {
		result = append(result, strings.Split(s, ",")...)
	}
```